Sources/AccountSDKIOSWeb/Lib/Extensions/UIFont+Custom.swift

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright © 2022 Schibsted.
+// Licensed under the terms of the MIT license. See LICENSE in the project root.
+//
+
+import UIKit
+
+extension UIFont {
+    static func preferredCustomFont(_ fontFamily: Inter, textStyle: UIFont.TextStyle) -> UIFont {
+        let systemDescriptor = UIFontDescriptor.preferredFontDescriptor(withTextStyle: textStyle)
+
+        let customFontDescriptor = UIFontDescriptor.init(
```

Comment:
Please don't use `.init`, just call the constructor normally:

```suggestion
        let customFontDescriptor = UIFontDescriptor(
```